src/core/agent/agent.types.ts

```diff
@@ -29,6 +29,8 @@ enum MiscRecordId {
   APP_BIOMETRY = "app-biometry",
   KERIA_NOTIFICATION_MARKER = "keria-notification-marker",
   APP_IDENTIFIER_VIEW_TYPE = "app-identifier-view-type",
+  KERI_URL = "keri-url",
+  KERI_BOOT_URL = "keri-boot-url",
```

Comment:
```
KERIA_CONNECT_URL = "keria-connect-url",
KERIA_BOOT_URL = "keria-boot-url",
```

Reply:
you only updated connect here, and not `KERIA_BOOT_URL = "keria-boot-url",`

Reply:
Yes